lib/vmdb/config.rb

```diff
@@ -162,9 +162,9 @@ def normalize_time(input)
       Time.parse(input.to_s).utc rescue nil unless input.nil?
     end
 
-    def config_mtime_from_db
+    def config_mtime_from_db(force_reload = true)
       # log_header = "[#{@name}]"
-      server = MiqServer.my_server(true)
+      server = MiqServer.my_server(force_reload)
       return Time.at(0) if server.nil?
 
       conf = server.configurations.select("updated_on").where(:typ => @name).first
```

Comment:
Also, this line is really why the my_server is reloaded.  We need the latest configurations record, not a cached one.